components/style/properties/helpers/animated_properties.mako.rs

```diff
@@ -2982,6 +2982,14 @@ impl Animatable for IntermediateSVGPaint {
         Ok(self.kind.compute_squared_distance(&other.kind)? +
             self.fallback.compute_squared_distance(&other.fallback)?)
     }
+
+    #[inline]
+    fn get_zero_value(&self) -> Option<Self> {
+        Some(IntermediateSVGPaint {
+            kind: option_try!(self.kind.get_zero_value()),
+            fallback: self.fallback.map(|v| v.get_zero_value().unwrap()),
```

Comment:
Err, that's not a `Result` returning method, so no need for `ok_or`:

```rust
self.fallback.and_then(|v| v.get_zero_value())
```

There is no reason to use `unwrap` if it can be avoided, this is code smell in general.